File: das_backend/src/main/java/ch/sbb/backend/logging/domain/LogLevel.java
package ch.sbb.backend.logging.domain;

public record LogLevel(String value) {

    public static final LogLevel TRACE = new LogLevel("TRACE");
    public static final LogLevel DEBUG = new LogLevel("DEBUG");
    public static final LogLevel INFO = new LogLevel("INFO");
    public static final LogLevel WARNING = new LogLevel("WARNING");
    public static final LogLevel ERROR = new LogLevel("ERROR");
    public static final LogLevel FATAL = new LogLevel("FATAL");
}

File: backend/src/main/kotlin/ch/sbb/backend/logging/domain/LogLevel.kt
package ch.sbb.backend.logging.domain

data class LogLevel(val value: String) {
    companion object {
        val TRACE = LogLevel("TRACE")
        val DEBUG = LogLevel("DEBUG")
        val INFO = LogLevel("INFO")
        val WARNING = LogLevel("WARNING")
        val ERROR = LogLevel("ERROR")
        val FATAL = LogLevel("FATAL")
    }
}
